packages/grafana-data/src/types/dataFrame.ts

```diff
@@ -57,6 +57,17 @@ export interface FieldConfig<TOptions extends object = any> {
   scopedVars?: ScopedVars;
 }
 
+export interface ValueLinkConfig {
```

Comment:
Alt name DataLinkValueContext , or DataLinkValueConfig  (just do not like the ValueLink prefix as it does not hint what where it is used )